SIS_import/docker-compose.yml

```diff
@@ -13,4 +13,4 @@ services:
     
```

Comment:
The docker file has a configuration that says ` ${HOME}/secrets/sis:/app/config/secrets` so I put the `.env` in ` home directory sis folder` but when I ran the script saying `docker compose up` it complained saying the `.env` is not in the project root directory. Just wondering why i see this problem ?

Reply:
I have reported this as a separate issue: #141